cspell.yml

```diff
@@ -7,6 +7,9 @@ ignorePaths:
   - package.json
   - benchmark/github-schema.graphql
   - benchmark/github-schema.json
+  - patches
+  - src/__benchmarks__/github-schema.json
+  - src/__benchmarks__/github-schema.graphql
```

Comment:
We could merge these two with the existing benchmark resource files into a shared folder